recipes/python-symengine/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+python setup.py install --symengine-dir=$PREFIX
```

Comment:
Thanks. Also, please add these arguments to your install command `--single-version-externally-managed --record record.txt`.